Userland/Libraries/LibJS/Runtime/JSONObject.cpp

```diff
@@ -383,7 +383,7 @@ DeprecatedString JSONObject::quote_json_string(DeprecatedString string)
             break;
```

Comment:
Commit title should just be "LibJS: " unless there's other cases you're planning on adding to this commit